composer.json

```diff
@@ -15,18 +15,25 @@
     }
   ],
   "require": {
-    "php": ">=5.5",
-    "symfony/symfony": "~2.4",
-    "doctrine/orm": "~2.2,>=2.2.3",
-    "doctrine/doctrine-bundle": "~1.2",
-    "symfony/monolog-bundle": "~2.4"
+    "php": ">=5.6",
+    "symfony/symfony": "~2.6|^3.0",
+    "doctrine/orm": "^2.2.3",
+    "doctrine/doctrine-bundle": "^1.2",
+    "symfony/monolog-bundle": "^2.4"
```

Comment:
It's almost the same as the tilde character, with the extra option to limit from a certain patch release. I see more OSS shifting to this notation (see NPM for instance, but also other PHP projects). I think it has useful benefits, for instance look at the `doctrine/orm` version.